.codeclimate.yml

```diff
@@ -109,14 +107,16 @@ plugins:
     enabled: true
     config:
       strings:
-      - FIXME
-      - BUG
-      - TODO
-      - todo
-      - dpm
-      - dsm
+        - FIXME
+        - BUG
+        - TODO
+        - todo
+        - dpm
+        - dsm
 # exclude these files/paths
 exclude_patterns:
   - "tests/"
   - "**.min.*"
   - "**/Gruntfile.js"
+  - "**.min.*"
```

Comment:
Are you sure?

Reply:
You don't plan on using minified js or css?

Reply:
its compiling the css, but not js. and scss files are reviewed by codeclimate. and none of the files are being compiled into `.min.` files at the moment.

Reply:
no idea if we plan to in the future

Reply:
Oh, this is codeclimate. Sorry I misread this as gitignore.

Ignore me and carry on.